poly/src/domain/radix2.rs

```diff
@@ -225,11 +228,13 @@ pub(crate) fn serial_radix2_fft<T: DomainCoeff<F>, F: FftField>(a: &mut [T], ome
     }
 
     let mut m = 1;
-    for _ in 0..log_n {
+    for _i in 1..=log_n {
```

Comment:
Hmm so the old code was subtly wrong?

Reply:
Or no, because we didn't use the index anywhere.

Reply:
It wasn't wrong, but writing the invariant to understand the code was awkward with the prior bounds